README.Rmd

```diff
@@ -32,34 +37,40 @@ likelihood inference, perform hypothesis testing with Satterthwaite
 adjusted degrees of freedom, and extract least square means estimates
 by using `emmeans`.
 
-## Main Features
+**Scope:**
 
-* Responses are assumed normally distributed.
-* Covariances:
-  * Structures: unstructured, Toeplitz, AR1, compound symmetry, and ante-dependence.
+* Continuous responses with normal (but potentially heteroscedastic) residuals.
+* Only marginal linear models without individual-level random effects are supported
+
+**Main Features:**
+
+* Flexible covariance specification:
+  * [Structures](https://openpharma.github.io/mmrm/main/articles/covariance.html): unstructured, Toeplitz, AR1, compound symmetry, and ante-dependence.
```

Comment:
can add now the spatial exponential as well :-)

Reply:
done